modules/ocf/manifests/systemuser.pp

```diff
@@ -0,0 +1,20 @@
+define ocf::systemuser(
+  $ensure = present,
+  $opts = {},
```

Comment:
Instead of doing this, you could make it only support the attributes we actually use and it'll look a little nicer on the user end of it since you won't have to specify `opts` when using it, but that's probably not a big deal and will lead to a bunch of params being repeated in this file. Anyway, there's a trade-off to be had there.